build_defs.bzl

```
@@ -118,6 +138,9 @@ compile_circuit = rule(
         "main": attr.label(mandatory = True, allow_single_file = [".circom"]),
         "includes": attr.string_list(),
         "deps": attr.label_list(providers = [CircomInfo]),
+        "generate_c": attr.bool(doc = "Compiles the circuit to C", default=True),
```

Formatter says you need to put space like below.

```suggestion
        "generate_c": attr.bool(doc = "Compiles the circuit to C", default = True),
```